src/student/forms.py

```diff
@@ -24,7 +23,33 @@ def clean(self):
         if is_student_exists:
             error_massage = "Student not added. Student with such phone_number and email is exists! Try again:"
             raise ValidationError(error_massage)
+        return self.cleaned_data
 
 
 class StudentEditForm(StudentBaseForm):
-    pass
+
+    def __init__(self, *args, **kwargs):
+        self.pk = kwargs.pop('instance', None)
+        super().__init__(*args, **kwargs)
+
+    def clean_email(self):
+        cleaned_data = super().clean()
+        email = cleaned_data.get('email').strip()
+
+        is_email_exists = Student.objects.filter(email=email).exclude(pk=self.pk).exists()
```

Comment:
An instance that is currently being updated is available as `sefl.instance`. So, you can get access to its id without extracting from params in constructor.